.github/maintainers-little-helper.yaml

```diff
@@ -1,198 +1,21 @@
-move-to-projects-for-labels-xored:
-  v1.14:
-    needs-backport/1.14:
-      project: "https://github.com/cilium/cilium/projects/255"
-      column: "Needs backport from main"
-    backport-pending/1.14:
-      project: "https://github.com/cilium/cilium/projects/255"
-      column: "Backport pending to v1.14"
-    backport-done/1.14:
-      project: "https://github.com/cilium/cilium/projects/255"
-      column: "Backport done to v1.14"
-  v1.13:
-    needs-backport/1.13:
-      project: "https://github.com/cilium/cilium/projects/257"
-      column: "Needs backport from main"
-    backport-pending/1.13:
-      project: "https://github.com/cilium/cilium/projects/257"
-      column: "Backport pending to v1.13"
-    backport-done/1.13:
-      project: "https://github.com/cilium/cilium/projects/257"
-      column: "Backport done to v1.13"
-  v1.12:
-    needs-backport/1.12:
-      project: "https://github.com/cilium/cilium/projects/256"
-      column: "Needs backport from main"
-    backport-pending/1.12:
-      project: "https://github.com/cilium/cilium/projects/256"
-      column: "Backport pending to v1.12"
-    backport-done/1.12:
-      project: "https://github.com/cilium/cilium/projects/256"
-      column: "Backport done to v1.12"
+project: "https://github.com/cilium/cilium/projects/239"
+column: "In progress"
+auto-label:
+  - "kind/backports"
+  - "backport/1.15"
 require-msgs-in-commit:
-  - regexpMsg: "(?m)^Signed-off-by:"
+  - msg: "Signed-off-by"
```

Comment:
I think this was changed accidentally?

Reply:
I copied this from the v1.14 branch, so yes this was unintentional.